internal/clients/readiness.go

```diff
@@ -148,9 +148,9 @@ func (c DefaultReadinessChecker) checkAlreadyCreatedClient(ctx context.Context,
 	defer cancel()
 	if err := client.IsReady(ctx); err != nil {
 		// Despite the error reason we still want to keep the client in the pending list to retry later.
-		c.logger.V(util.DebugLevel).Error(
-			err,
+		c.logger.V(util.DebugLevel).Info(
 			fmt.Sprintf("already created client for %q is not ready, moving to pending", client.BaseRootURL()),
```

Comment:
Maybe instead of embedding `client.BaseRootURL()` in the message, log it as a separate field. It will make logs easier to parse. WDYT?